tools/devices/lpc/stm32/mock/mock-stm32f4-27_29_37_39.xml

```diff
@@ -0,0 +1,1289 @@
+    <valid-device>mock-stm32f427aih</valid-device>
+    <valid-device>mock-stm32f427igh</valid-device>
+    <valid-device>mock-stm32f427igt</valid-device>
+    <valid-device>mock-stm32f427iih</valid-device>
+    <valid-device>mock-stm32f427iit</valid-device>
+    <valid-device>mock-stm32f427vgt</valid-device>
+    <valid-device>mock-stm32f427vit</valid-device>
+    <valid-device>mock-stm32f427zgt</valid-device>
+    <valid-device>mock-stm32f427zit</valid-device>
+    <valid-device>mock-stm32f429agh</valid-device>
+    <valid-device>mock-stm32f429aih</valid-device>
+    <valid-device>mock-stm32f429bet</valid-device>
+    <valid-device>mock-stm32f429bgt</valid-device>
+    <valid-device>mock-stm32f429bit</valid-device>
+    <valid-device>mock-stm32f429ieh</valid-device>
+    <valid-device>mock-stm32f429iet</valid-device>
+    <valid-device>mock-stm32f429igh</valid-device>
+    <valid-device>mock-stm32f429igt</valid-device>
+    <valid-device>mock-stm32f429iih</valid-device>
+    <valid-device>mock-stm32f429iit</valid-device>
+    <valid-device>mock-stm32f429neh</valid-device>
+    <valid-device>mock-stm32f429ngh</valid-device>
+    <valid-device>mock-stm32f429nih</valid-device>
+    <valid-device>mock-stm32f429vet</valid-device>
+    <valid-device>mock-stm32f429vgt</valid-device>
+    <valid-device>mock-stm32f429vit</valid-device>
+    <valid-device>mock-stm32f429zet</valid-device>
+    <valid-device>mock-stm32f429zgt</valid-device>
+    <valid-device>mock-stm32f429zgy</valid-device>
+    <valid-device>mock-stm32f429zit</valid-device>
+    <valid-device>mock-stm32f429ziy</valid-device>
+    <valid-device>mock-stm32f437aih</valid-device>
+    <valid-device>mock-stm32f437igh</valid-device>
+    <valid-device>mock-stm32f437igt</valid-device>
+    <valid-device>mock-stm32f437iih</valid-device>
+    <valid-device>mock-stm32f437iit</valid-device>
+    <valid-device>mock-stm32f437vgt</valid-device>
+    <valid-device>mock-stm32f437vit</valid-device>
+    <valid-device>mock-stm32f437zgt</valid-device>
+    <valid-device>mock-stm32f437zit</valid-device>
+    <valid-device>mock-stm32f439aih</valid-device>
+    <valid-device>mock-stm32f439bgt</valid-device>
+    <valid-device>mock-stm32f439bit</valid-device>
+    <valid-device>mock-stm32f439igh</valid-device>
+    <valid-device>mock-stm32f439igt</valid-device>
+    <valid-device>mock-stm32f439iih</valid-device>
+    <valid-device>mock-stm32f439iit</valid-device>
+    <valid-device>mock-stm32f439ngh</valid-device>
+    <valid-device>mock-stm32f439nih</valid-device>
+    <valid-device>mock-stm32f439vgt</valid-device>
+    <valid-device>mock-stm32f439vit</valid-device>
+    <valid-device>mock-stm32f439zgt</valid-device>
+    <valid-device>mock-stm32f439zgy</valid-device>
+    <valid-device>mock-stm32f439zit</valid-device>
+    <valid-device>mock-stm32f439ziy</valid-device>
+    <driver name="core" type="x86">
+      <memory device-size="e" name="flash" access="rx" start="0x8000000" size="524288"/>
```

Comment:
My intent was to minimize the changes to the mock file with the idea that it could, if desired in the future, be generated by modm's device generator with only a minimal change to the first two lines. So I figure it makes the most sense to keep everything else in the file the same so that GPIOs and timers and such are generated exactly as they would be otherwise.

Reply:
This file is also intended to be in the `modm-devices` submodule with the other device files rather than here; I just opted to put it here because we don't have a fork of that submodule and it wouldn't be pleasant to manage.